README.md

```diff
@@ -412,8 +435,7 @@ timestamp.source=wallclock
 
 Integration tests are implemented using JUnit, Gradle and Docker.
 
-To run them, you need:
-- Docker installed.
+To run them, you need Docker. Instruction available [here](https://docs.docker.com/engine/install/fedora/)
```

Comment:
I feel the instruction for installing Docker is easily googleable and it's not super important to include it in this connector's README. At least, the link should be to some more generic page instead of Fedora